tests/search/search-page.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { test, expect } from '@playwright/test';
+
+import { SearchPage } from './search-page';
+
+import { SortBar } from '../shared/sort-bar';
+
+let searchPage: SearchPage;
+
+test.describe('Search Page', () => {
+  test.describe.configure({ mode: 'serial' });
+
+  test('Go to search page and do a simple query', async ({ browser }) => {
+    const browserPage = await browser.newPage();
+    searchPage = new SearchPage(browserPage);
+
+    await searchPage.visit();
+    await searchPage.search('cats');
+  });
+
+  test.describe('Facets navigation', async () => {
+    test('Should display result count and different facet groups', async () => {
+      await searchPage.checkFacetGroups();
+    });
+  })
+
+  test('Navigate through different search view modes', async () => {
+    await searchPage.navigateThruInfiniteScrollerViewModes();
+  });
+
+  test.describe('Sorting results', async() => {
+    test('Sort filter - Weekly views in descending order', async () => {
```

Comment:
Question: Since the search page is doing the filtering, I'm not sure why the term "filter" is being used in test description here. The term doesn't seem to add understanding.

Suggest "Sort by weekly views, descending"  as clear and simple and short as possible